x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/eql/trial_license_complete_tier/eql_alert_suppression.ts

```diff
@@ -3181,7 +3179,7 @@ export default ({ getService }: FtrProviderContext) => {
         });
       });
 
-      it('does not suppress alerts outside of duration when query with 3 sequences', async () => {
+      it.skip('does not suppress alerts outside of duration when query with 3 sequences', async () => {
```

Comment:
We should add the comment referencing #202940 to this section: 
```suggestion
      // Failing: See https://github.com/elastic/kibana/issues/202940
      it.skip('does not suppress alerts outside of duration when query with 3 sequences', async () => {
```